test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/TestHelper.cs

```diff
@@ -226,14 +230,15 @@ protected void SetEnvironmentVariable(string key, string value)
             EnvironmentHelper.CustomEnvironmentVariables.Add(key, value);
         }
 
-        protected async Task AssertHttpSpan(
+        protected async Task AssertWebServerSpan(
```

Comment:
renamed why?

Reply:
To distinguish between server and client spans. Even though they are both "http," they are distinct integrations with very different expectations.

Reply:
Gotcha!